test/core/tsi/ssl_transport_security_test.cc

```diff
@@ -191,6 +191,18 @@ static void ssl_test_setup_handshakers(tsi_test_fixture* fixture) {
       TSI_OK);
 }
 
+static void check_ca_cert_subject(ssl_tsi_test_fixture* /*ssl_fixture*/,
+                                  const tsi_peer* peer) {
+  const tsi_peer_property* ca_cert_subject =
+      tsi_peer_get_property_by_name(peer, TSI_X509_CA_SUBJECT_PEER_PROPERTY);
+  ASSERT_NE(ca_cert_subject, nullptr);
+  const char* expected_match =
```

Comment:
nit: Please use `std::string` here instead of `const char*.

Reply:
`const char* expected match` is used through this test file, [example](https://github.com/grpc/grpc/blob/a4e3a22f5179fabc1670dcfa6d58e4adfaf7b02a/test/core/tsi/ssl_transport_security_test.cc#L322)